lib/vrl/tests/tests/internal/metadata_typedef.vrl

```diff
@@ -27,3 +27,12 @@ del(%a)
 .
 
 
+#set_metadata_field(., {})
+#set_metadata_field(.a, "string")
+#set_metadata_field(.b, 5)
+#.foo = type_def(get_metadata_field(.))
+#remove_metadata_field(.a)
+#.foo2 = type_def(get_metadata_field(.))
+#.
+
+
```

Comment:
This looks a bit out of place. Did you mean to rewrite these calls to use the new metadata operators?

Reply:
oops, I already _did_ rewrite them, just forgot to delete the previous code.

Reply:
oh, looking at this more, I think this came back from a merge conflict that I obviously resolved incorrectly